app/assets/javascripts/admin/addon/components/admin-embedding-host-form.gjs

```diff
@@ -0,0 +1,143 @@
+import Component from "@glimmer/component";
+import { inject as controller } from "@ember/controller";
+import { hash } from "@ember/helper";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import { isEmpty } from "@ember/utils";
+import BackButton from "discourse/components/back-button";
+import Form from "discourse/components/form";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { i18n } from "discourse-i18n";
+import AdminConfigAreaCard from "admin/components/admin-config-area-card";
+import CategoryChooser from "select-kit/components/category-chooser";
+import TagChooser from "select-kit/components/tag-chooser";
+import UserChooser from "select-kit/components/user-chooser";
+
+export default class AdminEmbeddingHostForm extends Component {
+  @service router;
+  @service site;
+  @service store;
+  @controller adminEmbedding;
+
+  get isUpdate() {
+    return this.args.host;
+  }
+
+  get header() {
+    return this.isUpdate
+      ? "admin.embedding.host_form.edit_header"
+      : "admin.embedding.host_form.add_header";
+  }
+
+  get formData() {
+    if (this.isUpdate) {
+      return {
+        host: this.args.host.host,
+        allowed_paths: this.args.host.allowed_paths,
+        category: this.args.host.category_id,
+        tags: this.args.host.tags,
+        user: isEmpty(this.args.host.user) ? null : [this.args.host.user],
+      };
+    } else {
+      return {};
+    }
+  }
+
+  @action
+  async save(data) {
+    const host = this.args.host || this.store.createRecord("embeddable-host");
+
+    try {
+      await host.save({
+        ...data,
+        user: data.user?.at(0),
```

Comment:
Bit strange, could you use `this.currentUser`?

Reply:
Or is there something special about user at position 0?

Reply:
Its this one

<img width="460" alt="Screenshot 2024-12-19 at 4 44 39 PM" src="https://github.com/user-attachments/assets/0ade109b-10b8-4abe-b866-c2fa552911be" />

Even with maximum 1 it returns array:
```
                  <UserChooser
                    @value={{field.value}}
                    @onChange={{field.set}}
                    @options={{hash maximum=1 excludeCurrentUser=false}}
                    class="admin-embedding-host-form__post_author"
                  />
```

Reply:
OK ty I didn't realise :)